doc/create-cloud-resources.rst

```diff
@@ -37,10 +37,7 @@ connect your credit card or other payment method to your google cloud account.
    commands to Google Cloud and lets you do things like create and delete
    clusters.
 
-   - Go to the `gcloud downloads page <https://cloud.google.com/sdk/downloads>`_
-     to **download and install the gcloud SDK**.
-   - See the `gcloud documentation <https://cloud.google.com/sdk/>`_ for
-     more information on the gcloud SDK.
+   - Use Google Cloud Shell which will give you access to the ``gcloud`` command-line tool.
```

Comment:
Thanks @aculich. I agree that this approach makes sense and I would add a second sentence here on where to find on the nav bar for Google Cloud.

I would put the removed text in a directive for advanced users directly below.

Line 36 needs to be tweaked a bit. In fact, it may be better to break out line 41 as it's own step (new Step 4).